packages/svelte/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@wagmi/svelte",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build && npm run package",
+    "preview": "vite preview",
+    "package": "svelte-kit sync && svelte-package && publint",
+    "prepublishOnly": "npm run package",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "lint": "eslint . && prettier --check .",
+    "format": "prettier --write .",
+    "test:unit": "vitest",
+    "test": "pnpm run test:unit -- --run --watch"
+  },
+  "files": ["dist", "!dist/**/*.test.*", "!dist/**/*.spec.*"],
+  "sideEffects": ["**/*.css"],
+  "svelte": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
+    },
+    "./chains": {
+      "types": "./dist/exports/chains.d.ts",
+      "default": "./dist/exports/chains.js"
+    }
+  },
+  "peerDependencies": {
+    "@tanstack/svelte-query": "https://pkg.pr.new/@tanstack/svelte-query@ccce0b8",
+    "viem": "2.x",
+    "svelte": "^5.0.0"
+  },
+  "devDependencies": {
```

Comment:
Would like to avoid Prettier and ESLint if [Biome support](https://biomejs.dev/internals/language-support) is good enough.

Reply:
Mainly resolved in https://github.com/wevm/wagmi/pull/4365/commits/2917d4254af926f51af667a0b8e801b9adaf0854 with minor changes in https://github.com/wevm/wagmi/pull/4365/commits/9d2c30adddd6fa66101ceb3985531066537cf71b